 nut (2.7.2-1.netgear2) stable; urgency=low
 .
   * Disable riello_usb, since it hijacks generic Cypress serial adapters.
Author: Justin Maggard <justin.maggard@netgear.com>
---

--- nut-2.7.2.orig/scripts/udev/nut-usbups.rules.in
+++ nut-2.7.2/scripts/udev/nut-usbups.rules.in
@@ -51,10 +51,6 @@ ATTR{idVendor}=="0463", ATTR{idProduct}=
 #  TS Shara UPSes  - nutdrv_qx
 ATTR{idVendor}=="0483", ATTR{idProduct}=="0035", MODE="664", GROUP="@RUN_AS_GROUP@"
 
-# Riello (Cypress Semiconductor Corp.)
-#  various models  - riello_usb
-ATTR{idVendor}=="04b4", ATTR{idProduct}=="5500", MODE="664", GROUP="@RUN_AS_GROUP@"
-
 # Minibox
 #  openUPS Intelligent UPS (minimum required firmware 1.4)  - usbhid-ups
 ATTR{idVendor}=="04d8", ATTR{idProduct}=="d004", MODE="664", GROUP="@RUN_AS_GROUP@"
--- nut-2.7.2.orig/tools/nut-usbinfo.pl
+++ nut-2.7.2/tools/nut-usbinfo.pl
@@ -293,7 +293,7 @@ sub find_usbdevs
 				$driver="nutdrv_atcl_usb";
 			}
 			elsif ($nameFile eq "riello_usb.c") {
-				$driver="riello_usb";
+				next;
 			}
 			elsif ($nameFile eq "nutdrv_qx.c") {
 				$driver="nutdrv_qx";
